Module_2.js

```diff
@@ -0,0 +1,132 @@
+var timeKeeper = ( function () {
+
+  var date = new Date();
+  var MONTHS = new Array( 12 );

```

Comment:
When declaring arrays, it's preferred to just use array literals:

    var MONTHS = ['January', 'February'...];